src/pages/jaeyoungLee/Main/Main.js

```diff
@@ -0,0 +1,255 @@
+import React from 'react';
+import CommentComponent from './CommentComponent/CommentComponent';
+import './Main.scss';
+import explore from '../../../images/jaeyoungLee/Main/explore.jpg';
+import heart from '../../../images/jaeyoungLee/Main/heart.jpg';
+import profile from '../../../images/jaeyoungLee/Main/profile.jpg';
+import heungminSon from '../../../images/jaeyoungLee/Main/손흥민.jpg';
+class MainJaeyoung  extends React.Component {
+    constructor () {
+        super();
+        this.state = {
+            commentValue : '',
+            commentList : [],
+        }
+    }
+
+    getCommentValue = (e) => {
+        const { value } = e.target;
+        this.setState({
+            commentValue : value
+        })
+    }
+
+    clickButton = () => {
+        const { commentList,commentValue } = this.state;
+        this.setState({
+            commentList : [...commentList, commentValue],
+            commentValue : ''
+        })
+    }
+
+    enterKey = (e) => {
```

Comment:
```suggestion
    inputKeyPress= (e) => {
```
같은 이름은 어떨까요? 메소드가 많아지면 헷갈릴 수 있을 거 같습니다.